www/core/test/list_test.xml

```diff
@@ -1,7 +1,7 @@
 <extensionset name="Joomla Core Test Updateserver" description="The Joomla Core Updateserver for Tests of Alpha, Beta und RC Releases">
```

Comment:
done btw you can also do that using the edit tool from github as you have write access to that repo.